app/crud.py

```
@@ -0,0 +1,63 @@
+from sqlalchemy.future import select
+
+from .models import ToDo
+
+
+async def create_todo(db: AsyncSession, todo: ToDo):
+    try:
+        db.add(todo)
+        await db.commit()
+        await db.refresh(todo)
+        return todo
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)